tests/db/envelop_test.py

```diff
@@ -0,0 +1,13 @@
+# encoding=utf-8
+
+import unittest
+
+from alot.db.envelope import Envelope
+
+
+class TestEnvelopeMethods(unittest.TestCase):
+
+    def test_setitem(self):
+        e = Envelope()
+        e['Subject'] = u'sm\xf8rebr\xf8d'
```

Comment:
Do you have an idea about what to write in the comment. I just copied that from the doctest.